internal/js/modules/k6/browser/common/js/injected_script.js

```diff
@@ -225,6 +1478,7 @@ class InjectedScript {
       css: new CSSQueryEngine(),
       text: new TextQueryEngine(),
       xpath: new XPathQueryEngine(),
+      'internal:role': createRoleEngine(true),
```

Comment:
Sticking with how Playwright has labelled it. There is also a selector without `internal` which we could consider adding. It is so that users can query without the `getByRole` API, and directly do something like: `page.locaotr('role=button');`.

Reply:
Yeah, I think that would be nice, specially because I think `internal:role` is particular to Playwright, while the other looks more like "the standard"?

Is there any reason why not doing so on this PR? 
Does it require any changes beyond also adding that pair of keyword and engine? 🤔

Reply:
> Is there any reason why not doing so on this PR?

Mainly because I wanted to narrow the focus to working with the dedicated APIs that we have implemented `getByRole` instead of allowing for the manual creation of role selectors. It can get quite complicated to manually create a role selector with options. So it's an ease of use thing mainly.

> Does it require any changes beyond also adding that pair of keyword and engine? 🤔

That's basically it, add another case for `role` without `internal:`.